test/utilities/extensionUtils.ts

```diff
@@ -20,13 +20,21 @@ export async function findExtensionInRunningExtensionsList(
   // This function assumes the Extensions list was opened.
 
   // Close the panel and clear notifications so we can see as many of the running extensions as we can.
-  await runCommandFromCommandPrompt(workbench, 'View: Close Panel', 1);
+  try {
+    await runCommandFromCommandPrompt(workbench, 'View: Close Panel', 1);
+  }
+  catch {
+    // Close the command prompt by hitting the Escape key
+    await browser.keys(['Escape']);
+    log('No panel to close - command not found');
+  }
```

Comment:
Try/catch block is needed because "View: Close Panel" is not available in the command palette if the Output panel is already closed.